tests/IterableCodeExtractorTest.php

```diff
@@ -2,16 +2,16 @@
 
 namespace WP_CLI\I18n\Tests;
 
-use PHPUnit_Framework_TestCase;
 use WP_CLI\I18n\IterableCodeExtractor;
 use WP_CLI\Utils;
+use Yoast\PHPUnitPolyfills\TestCases\TestCase;
 
-class IterableCodeExtractorTest extends PHPUnit_Framework_TestCase {
+class IterableCodeExtractorTest extends TestCase {
 
 	/** @var string A path files are located */
 	private static $base;
 
-	public function setUp() {
+	public function set_up() {
```

Comment:
@jenkoian Can you undo the changes here? The tests will fail locally but pass on CI.

I think this needs to be fixed in a separate PR.

Reply:
Have used the `WP_CLI\Tests\TestCase` as mentioned above and changed this back as you suggested. Was that all?